chalice/deploy/deployer.py

```diff
@@ -141,15 +141,29 @@ def confirm(self, text, default=False, abort=False):
         # type: (str, bool, bool) -> bool
         return default
 
+    def Abort(self):
```

Comment:
Well, after thinking about this, I'm not even sure if abort() make sense here.  This interface is just about prompting.  Aborting doesn't really make sense here.  Does this need to be an explicit part of this interface?  Can we not just raise an exception?